individual.h

```diff
@@ -25,6 +25,42 @@ class individual
 public:
 
     individual(ind_param i_p = {});
+    individual(individual&&) = default;
```

Comment:
I don't understand this

Reply:
this is the default assignment constructor, complicated stuff, but basically allows you to build an object from a reference, mainly necessary for std:: algorithms

Reply:
having a unique_ptr deletes it from the class (it is usually implicitly compiled) so we need to declare it explicitly